ui/packages/app/src/project_setting/ProjectInfoSetting.js

```diff
@@ -0,0 +1,29 @@
+import React, { useContext } from "react";
+import ProjectInfoForm from "./project_info/ProjectInfoForm";
+import { ProjectFormContextProvider } from "./form/context";
+import { ProjectsContext } from "@caraml-dev/ui-lib";
+import { Project } from "./form/project";
+import { EuiLoadingChart, EuiTextAlign } from "@elastic/eui";
+
+const ProjectInfoSetting = () => {
+  const { currentProject, refresh } = useContext(ProjectsContext);
+
+  return (
+    <>
+      {!currentProject ? (
+        <EuiTextAlign textAlign="center">
+          <EuiLoadingChart size="xl" mono />
+        </EuiTextAlign>
+      ) : (
+        <ProjectFormContextProvider project={currentProject}>
+          <ProjectInfoForm
+            originalProject={Project.from(currentProject)}
```

Comment:
Not too important, but I would prefer passing in a copy of `currentProject` (i.e. `Project.from(currentProject)` as a prop to the context provider) and the original `currentProject` as a prop of `ProjectInfoForm` instead (the reverse of what you're doing), because we aren't exactly mutating `currentProject` (as `originalProject`) in  `ProjectInfoForm` but we'll be mutating the project that's provided by the `ProjectFormContextProvider` (so it'll be nice not to update the original one provided by `ProjectsContext`).